Add Dark Theme 1.0

daniel8lopes opened the pull request.

daniel8lopes committed:
Add Dark Mode Theme 1.0

gsferreira commented on Themes/DarkMode.scss:
```diff
@@ -0,0 +1,177 @@
+// Color system
+//
+
+$white:    #fff !default;
```
remove the !default tag from all variables.

daniel8lopes replied:
Done

gsferreira commented on Themes/DarkMode.scss:
```diff
@@ -0,0 +1,177 @@
+
+$font-family-sans-serif:      "Lato", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
+
+//$font-size-base:              0.9375rem !default;
```
If we don't need it, please remove.

daniel8lopes replied:
Done

daniel8lopes committed:
Removed commented styles and !default tag

daniel8lopes committed:
Added screenshots and organized folders

daniel8lopes committed:
Changed readme from txt to .me and added text to dark theme

daniel8lopes committed:
Rename readme.me.txt to readme.me

daniel8lopes committed:
Rename readme.me to readme.md

daniel8lopes committed:
Update readme.md

daniel8lopes committed:
Update readme.md

daniel8lopes committed:
Update readme.md

daniel8lopes committed:
added "NavyBlue" readme content

daniel8lopes committed:
Merge branch 'daniel8lopes-patch-1' of https://github.com/OMNIALowCode/omnia3-samples into daniel8lopes-patch-1

daniel8lopes committed:
Rename readme.md.txt to readme.md

daniel8lopes committed:
Update readme.md

daniel8lopes committed:
Added comments to the scss (were removed by mistake)

daniel8lopes committed:
Merge branch 'daniel8lopes-patch-1' of https://github.com/OMNIALowCode/omnia3-samples into daniel8lopes-patch-1

daniel8lopes committed:
Added imsg folder and theme preview images inside it

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
New preview imgs

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Edited Dark css and one imgs

Corrected secondary theme color from gray-700 to gray-600 to increase contrast between borders and background

daniel8lopes committed:
Added comments where common edits may occur

Colors and type of charts

daniel8lopes committed:
Added OMNIA header (as comment)

Added OMNIA header (as comments) with some pointers to help devs understand what's going on

daniel8lopes committed:
Added "ORIGINAL" theme

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Update README.md

daniel8lopes committed:
Added ListView Custom Styles to all themes
